### PR TITLE
ADM-712:[docs] docs: optimize document content

### PR DESCRIPTION
**Pull request by yulongcai**
## Summary

optimize document content

## Before

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## After

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## Note

Only the contents of the document are added

### CODE REVIEW

File: docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+title: Spike calling jira with graphQL API
 description: Jira graphQL API about replacing existing REST API.
 ---
 
-# Spike -- Consider the current API call for Jira and get the replacement by graphQL API.
-
 ## Current situation
 
-The existing jira api are all rest style, and the rest api has the problem of over-fetching or multiple requests.
+The existing jira API are all REST API, and the REST API has the problem of over-fetching or multiple requests.
```
Comment (mikeyangyun) [inline docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx:8]: change `jira` -> `Jira`,`REST` -> `RESTFUL`

File: docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
 ## Solutions
-Jira provides the functionality of the graphQL api, which provides us with an api to get a variety of data at once and return only the data we need.
+Jira provides the functionality of the graphQL API, which provides us with an API to get a variety of data at once and return only the data we need.
 
 ## Example
 
-### 1.1 `/boards/{boardType}/verify`
-jira interface `path = "/rest/agile/1.0/board/{boardId}"`, only projectKey is used in the logic。
+
+### 一、For heartbeat's API `/boards/{boardType}/verify`
```
Comment (mikeyangyun) [inline docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx:23]: change `一` to english expression

File: docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx [hunk 3]
```diff
@@ -133,15 +134,25 @@ Query Variables
...
-There are three other interfaces to the rest api for jira that do not support graphQL replacement.
+There are three other interfaces to the REST API for jira that do not support graphQL API replacement. as follows:
+
+1. `/rest/internal/2/issue/{jiraCardKey}/activityfeed`, no query history card record was found in the current data.
+
+2. `/rest/api/2/issue/createmeta`, no way to query.
+
+3. `/rest/api/2/project/{projectIdOrKey}`, the graphQL API returns a style of `TEAM_MANAGED_PROJECT` and `COMPANY_MANAGED_PROJECT`，not in the style we need.
+
+## Test jira graphQL API
```
Comment (mikeyangyun) [inline docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx:150]: change `jira` -> Jira

File: docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx [hunk 4]
```diff
@@ -42,13 +42,14 @@ Query Variables
 }
 ```
 
-### 1.2 `jira interface path = "/boards /{boardType}" and "/boards /{boardType}/info"`
-1. `/rest/agile/1.0/board/{boardId}/configuration`, obtain the board id, name attribute, column name, and status id。
-2. `/rest/api/2/status/{statusNum}`, get the untranslatedName of status and the key and name of category。
-3. `/rest/agile/1.0/board/{boardId}/issue? maxResults={queryCount}&startAt={startAt}&jql={jql}`, get all cards on the board, but some card fields are missing
+### 二、For heartbeat's API `/boards /{boardType}` and `/boards /{boardType}/info`
```
Comment (mikeyangyun) [inline docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx:45]: change `二` to english expression

File: docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx (hunk 1 above)
Comment (yulongcai) [reply docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx]: done

File: docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx (hunk 2 above)
Comment (yulongcai) [reply docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx]: done

File: docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx (hunk 3 above)
Comment (yulongcai) [reply docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx]: done

File: docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx (hunk 4 above)
Comment (yulongcai) [reply docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx]: done

File: docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx [hunk 5]
```diff
@@ -75,9 +74,9 @@ Query Variables
   "boardId": "2"
 }
 ```
-As you can see, the rest api needs to call the interface twice to get the required value, and graphQL only needs to call once
+As we can see, the REST API needs to call the interface twice to get the required value, and graphQL API only needs to call once.
```
Comment (mikeyangyun) [inline docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx:77]: change `REST` -> `RESTFUL`

File: docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx [hunk 6]
```diff
@@ -133,15 +132,26 @@ Query Variables
...
 ```
-With this graphQL replacement, the fields required in the project are the statusCategoryChangeDate and customFields, which require further investigation
+With this graphQL API replacement, the fields required in the project are the `statusCategoryChangeDate` and `customFields`, which require further investigation.
 
-As you can see from graphQL above, these three rest apis can be replaced by a single graphQL api, simply by modifying the schema
+As you can see from graphQL API above, these three rest apis can be replaced by a single graphQL API, simply by modifying the schema.
 
 ## Question
-There are three other interfaces to the rest api for jira that do not support graphQL replacement.
+There are three other interfaces to the REST API for jira that do not support graphQL API replacement. as follows:
```
Comment (mikeyangyun) [inline docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx:140]: change `REST` -> `RESTFUL`

File: docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx (hunk 5 above)
Comment (yulongcai) [reply docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx:77]: done

File: docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx (hunk 6 above)
Comment (yulongcai) [reply docs/src/content/docs/en/spikes/tech-spikes-jira-graphql-api-about-replacing-existing-rest-api.mdx]: done

**Review by mikeyangyun — APPROVED** [review]

**Pull request merged**